<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #393A4B</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="8" column="4">
     <widget class="QPushButton" name="MemGet">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Tibetan Machine Uni</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #D38830;
	border: 1px solid gray;
	padding: 5px;
	color: black;
border-radius: 5px
}

QPushButton:pressed {
	background-color: #AF7128;
	border: 1px solid gray;
	padding: 5px;
	color: black;
border-radius: 5px
}
</string>
      </property>
      <property name="text">
       <string>M</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="Cubed">
      <property name="text">
       <string>x³</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="Sin">
      <property name="text">
       <string>sin</string>
      </property>
     </widget>
    </item>
    <item row="8" column="2">
     <widget class="QPushButton" name="Button3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Tibetan Machine Uni</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: 5px;
	color: black;
	border-radius: 5px;
}

QPushButton:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding: 5px;
	color: black;
	border-radius: 5px;
}
</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QPushButton" name="Button6">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Tibetan Machine Uni</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: 5px;
	color: black;
	border-radius: 5px;
}

QPushButton:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding: 5px;
	color: black;
	border-radius: 5px;
}
</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="Perc">
      <property name="text">
       <string>%</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="Deg">
      <property name="text">
       <string>deg</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QPushButton" name="Button5">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Tibetan Machine Uni</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: 5px;
	color: black;
	border-radius: 5px;
}

QPushButton:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding: 5px;
	color: black;
	border-radius: 5px;
}
</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="Mod">
      <property name="text">
       <string>mod</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QPushButton" name="Factorial">
      <property name="text">
       <string>!</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="Exp">
      <property name="text">
       <string>e</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="ArcSin">
      <property name="text">
       <string>sin-1</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QPushButton" name="Clear">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Tibetan Machine Uni</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #B74129;
	border: 1px solid gray;
	padding: 5px;
	color: gold;
	border-radius: 5px;
}

QPushButton:pressed {
	background-color: #9E311B;
	border: 1px solid gray;
	padding: 5px;
	color: black;
	border-radius: 5px;
}
</string>
      </property>
      <property name="text">
       <string>AC</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="Pi">
      <property name="text">
       <string>π</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="Squared">
      <property name="text">
       <string>x²</string>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QPushButton" name="Divide">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Tibetan Machine Uni</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #7E2F40;
	border: 1px solid gray;
	padding: 5px;
	color: white;
	border-radius: 5px;
}

QPushButton:pressed {
	background-color: #6C2937;
	border: 1px solid gray;
	padding: 5px;
	color: white;
	border-radius: 5px;
}
</string>
      </property>
      <property name="text">
       <string>÷</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QPushButton" name="Button2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Tibetan Machine Uni</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: 5px;
	color: black;
	border-radius: 5px;
}

QPushButton:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding: 5px;
	color: black;
	border-radius: 5px;
}
</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="9" column="2">
     <widget class="QPushButton" name="ChangeSign">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Tibetan Machine Uni</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: 5px;
	color: black;
	border-radius: 5px;
}

QPushButton:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding: 5px;
	color: black;
	border-radius: 5px;
}
</string>
      </property>
      <property name="text">
       <string>+|-</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QPushButton" name="Button0">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Tibetan Machine Uni</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: 5px;
	color: black;
	border-radius: 5px;
}

QPushButton:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding: 5px;
	color: black;
	border-radius: 5px;
}
</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="Button8">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Tibetan Machine Uni</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: 5px;
	color: black;
	border-radius: 5px;
}

QPushButton:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding: 5px;
	color: black;
	border-radius: 5px;
}
</string>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QPushButton" name="Button9">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Tibetan Machine Uni</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: 5px;
	color: black;
	border-radius: 5px;
}

QPushButton:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding: 5px;
	color: black;
	border-radius: 5px;
}
</string>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="ArcCos">
      <property name="text">
       <string>cos-1</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="Log">
      <property name="text">
       <string>log</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="Ceil">
      <property name="text">
       <string>⌈x⌉</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="Cos">
      <property name="text">
       <string>cos</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="Tan">
      <property name="text">
       <string>tan</string>
      </property>
     </widget>
    </item>
    <item row="7" column="4">
     <widget class="QPushButton" name="MemClear">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Tibetan Machine Uni</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #D38830;
	border: 1px solid gray;
	padding: 5px;
	color: black;
border-radius: 5px
}

QPushButton:pressed {
	background-color: #AF7128;
	border: 1px solid gray;
	padding: 5px;
	color: black;
border-radius: 5px
}
</string>
      </property>
      <property name="text">
       <string>M-</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="Rad">
      <property name="text">
       <string>rad</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="Button7">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Tibetan Machine Uni</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: 5px;
	color: black;
	border-radius: 5px;
}

QPushButton:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding: 5px;
	color: black;
	border-radius: 5px;
}
</string>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="7" column="3">
     <widget class="QPushButton" name="Multiply">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Tibetan Machine Uni</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #7E2F40;
	border: 1px solid gray;
	padding: 5px;
	color: white;
	border-radius: 5px;
}

QPushButton:pressed {
	background-color: #6C2937;
	border: 1px solid gray;
	padding: 5px;
	color: white;
	border-radius: 5px;
}
</string>
      </property>
      <property name="text">
       <string>x</string>
      </property>
     </widget>
    </item>
    <item row="9" column="4">
     <widget class="QPushButton" name="Equals">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Tibetan Machine Uni</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #258324;
	border: 1px solid gray;
	padding: 5px;
	color: white;
	border-radius: 5px;
}

QPushButton:pressed {
	background-color: #1D681C;
	border: 1px solid gray;
	padding: 5px;
	color: white;
 border-radius: 5px;
}
</string>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QPushButton" name="Button1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Tibetan Machine Uni</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: 5px;
	color: black;
	border-radius: 5px;
}

QPushButton:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding: 5px;
	color: black;
	border-radius: 5px;
}
</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="5">
     <widget class="QLineEdit" name="Display">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	background-color: #ABA982;
	border: 1px solid gray;
	color: #black;
	border-radius: 5px;
}</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="ArcTan">
      <property name="text">
       <string>tan-1</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="SquareRoot">
      <property name="text">
       <string>√</string>
      </property>
     </widget>
    </item>
    <item row="6" column="4">
     <widget class="QPushButton" name="MemAdd">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Tibetan Machine Uni</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #D38830;
	border: 1px solid gray;
	padding: 5px;
	color: black;
border-radius: 5px
}

QPushButton:pressed {
	background-color: #AF7128;
	border: 1px solid gray;
	padding: 5px;
	color: black;
border-radius: 5px
}
</string>
      </property>
      <property name="text">
       <string>M+</string>
      </property>
     </widget>
    </item>
    <item row="8" column="3">
     <widget class="QPushButton" name="Add">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Tibetan Machine Uni</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #7E2F40;
	border: 1px solid gray;
	padding: 5px;
	color: white;
	border-radius: 5px;
}

QPushButton:pressed {
	background-color: #6C2937;
	border: 1px solid gray;
	padding: 5px;
	color: white;
	border-radius: 5px;
}
</string>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="ComplementOfBase">
      <property name="text">
       <string>compl</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="CubeRoot">
      <property name="text">
       <string>3√a</string>
      </property>
     </widget>
    </item>
    <item row="9" column="3">
     <widget class="QPushButton" name="Subtract">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Tibetan Machine Uni</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #7E2F40;
	border: 1px solid gray;
	padding: 5px;
	color: white;
	border-radius: 5px;
}

QPushButton:pressed {
	background-color: #6C2937;
	border: 1px solid gray;
	padding: 5px;
	color: white;
	border-radius: 5px;
}
</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QPushButton" name="Button4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Tibetan Machine Uni</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: 5px;
	color: black;
	border-radius: 5px;
}

QPushButton:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding: 5px;
	color: black;
	border-radius: 5px;
}
</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QPushButton" name="Off">
      <property name="styleSheet">
       <string notr="true">background-color: grey</string>
      </property>
      <property name="text">
       <string>I/O</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>442</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Latin Modern Mono Caps</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white</string>
   </property>
   <widget class="QMenu" name="menuScientifica">
    <property name="font">
     <font>
      <family>STIXIntegralsD</family>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Calculadora Científica</string>
    </property>
   </widget>
   <addaction name="menuScientifica"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
